searchcore/src/tests/proton/documentdb/document_subdbs/document_subdbs_test.cpp

```diff
@@ -557,6 +557,29 @@ TEST_F("require that attribute manager can be reconfigured", SearchableFixture)
     requireThatAttributeManagerCanBeReconfigured(f);
 }
 
+TEST_F("require that subdb reflect retirement", FastAccessFixture)
```

Comment:
We should also add a test that verifies that the new compaction config is applied to an attribute vector.

Reply:
Done, both documentmetastore and attributes.